code/frameworks/nextjs/src/nextImport/webpack.ts

```diff
@@ -20,7 +21,7 @@ export function configureNextImport(baseConfig: WebpackConfig) {
     );
   }
 
-  if (!isNext12) {
+  if (isNext12 && isNextVersionSmallerThan12dot2) {
```

Comment:
What about Next > 12. for these versions the package has to be ignored as well

Reply:
```suggestion
  if (isNext12 && isNextVersionSmallerThan12dot2 || isNextVersionGreaterThan12) {
```

Reply:
Doesn't `isNext12` handle that?

Reply:
Sorry, forgot some brackets

Reply:
So for every next version, except of version ^12.2.0, we would like to initialize the ignore plugin

Reply:
Ahh, right. 🤦‍♂️ I think I kept mentally considering the negation (`!`) still there, even though I removed it.

Reply:
Actually, I think I found a simpler version:

```js
if (!isNext12 || isNextVersionSmallerThan12dot2) {
```

Reply:
Or `! semver.satisfies(nextJSVersion, '^12.2.0')`